src/cdk/drag-drop/drag-utils.ts

```diff
@@ -51,6 +57,35 @@ export function transferArrayItem<T = any>(currentArray: T[],
   }
 }
 
+/**
+ * Copies item from one array to another. Modifying transferArrayItem from cdk utils found here
+ *
+ * https://github.com/angular/material2/blob/master/src/cdk/drag-drop/drag-utils.ts
+ *
+ * @param currentArray Array from which to clone the item.
```

Comment:
I think the wording should avoid talking about "cloning", because it sounds like it would clone the item as it's copying it.